release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1269,6 +1269,13 @@ Optionally, you can now override the default evaluation intervals for all polici
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-configuring-pgt-compliance-eval-timeouts_ztp-advanced-policy-config[Configuring policy compliance evaluation timeouts for PolicyGenTemplate CRs].
 
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries"]
+==== Configuring the hub cluster to use unauthenticated registries
+
+This release supports the use of unauthenticated registries when configuring the hub cluster. Registries that don't require authentication can be listed under `spec.unauthenticatedRegistries` in the `AgentServiceConfig` resource. 
+Any registry on this list is not required to have an entry in the pull secret validated by the `assisted-service`.
+For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc#ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_ztp-preparing-the-hub-cluster[Configuring the hub cluster to use unauthenticated registries].
```

Comment:
I assume this module isn't present in the repo yet??

Reply:
Yes, it's a new module that is part of the content for TELCODOCS-1050.